src/components/CartCard.jsx

```diff
@@ -0,0 +1,83 @@
+import { useNavigate } from "react-router-dom";
+import { useCart } from "../contexts";
+
+export const CartCard = ({ productDetails }) => {
+  const { _id, title, price, discount, discountPrice, imageUrl, quantity } =
+    productDetails;
+  const {
+    cartState: { wishlist },
```

Comment:
you can destructure cartState and then provide as value from the context itself so that you don't have to destructure it in every component

Reply:
Makes sense, made the change.